### ISSUE TITLE
Learning Resource Type Vocabualry

### ISSUE THREAD

**Issue by HughP** [dcmi/LRMI_Vocabs#43]
Greetings,

I have been going over the Learning Resource Type Vocabulary. I am wondering how I might categorize the following items:

* syllabi
* handouts

Additionally I have found web resources (by AAP PreK-12 Learning Group) which suggest the following list of items...
![screen shot 2016-04-08 at 11 10 22 am](https://cloud.githubusercontent.com/assets/40230/14419784/f9837270-ff7c-11e5-8f70-93e930c33df1.jpg)

http://www.slideshare.net/edpublishers/lrmi-using-education-metadata-to-support-learning?qid=10295851-7c6f-4032-9736-cbd71093bf8d&v=&b=&from_search=2
 What is one to do with conflicting list? Is the list of items in this vocabulary "open to the needs of the institution"? (of course at the cost of interoperability).

**Comment by jgoodell2** [comment]
Hugh;
I think the AAP list was based on earlier LRMI pilots and contains some overlaps like "audio" and "podcast" and mixing type of media and purpose of resource. The U.S. multi-state tagging initiative and CEDS defined another list based on purpose and level of granularity. The tagging initiative decided that the kind of media needed to be separate from the kind of use and granularity. E.g. The resource is either a Course, Unit, Lesson, or Learning Activity (granularity).  A Formative Assessment might be designed as a Learning Activity within a Lesson in which case both tags could be used, or an independent resource just as Formative Assessment. ...not perfect, but widely used. 


Common Education Data Standards (CEDS) 

  
|  
|   
|   
|   |    |

   |

  |
|  
|   |  
Common Education Data Standards (CEDS)
 The Common Education Data Standards (CEDS) project is a national collaborative effort to develop voluntary, comm...  |   |

  |

  |

 
 
Learning Resource Type
 DefinitionThe predominate type or kind characterizing the learning resource.Usage NotesA community of SEAs have agreed to use a limited set of Learning Resource Type options when tagging learning resources in shared resource repositories. The options defined for this element align with those common tagging specifications.

| Option Set |
|  
|

 |
| Alternate Assessment | An assessment that is used to evaluate the performance of students who are unable to participate in general state assessments even with accommodations. | AlternateAssessment |
| Assessment Item | A specific prompt, that defines a question or protocol for a measurable activity that triggers a response from a person used to determine whether the person has mastered a learning objective. | AssessmentItem |
| Course | A series of units and lessons used to teach the skills and knowledge required by its curriculum. | Course |
| Demonstration/Simulation | An imitation or modeling of a real-world process. | DemonstrationSimulation |
| Educator/Curriculum Guide | A document that defines what concepts should be taught and/or how a concept should be taught effectively. | EducatorCurriculumGuide |
| Formative assessment | A process used by teachers and students during instruction that provides feedback to adjust ongoing teaching and learning to improve students’ achievement of intended instructional outcomes. (CCSSO FAST SCASS, 2006) | FormativeAssessment |
| Images/Visuals | Visual media, including but not limited to pictures, graphics, diagrams, figures, illustrations, charts, and maps. | ImagesVisuals |
| Interim/Summative Assessment | An assessment instrument used to evaluate student learning at the end of an instructional unit by comparing it against some standard or benchmark. A learning resource of this typemay be an "assessment form," i.e. one instance of the assessment instrument that can equate scores with another instance of the same assessment. | InterimSummativeAssessment |
| Learning Activity | Activities engaged in by the learner for the purpose of acquiring certain skills, concepts, or knowledge, whether guided by an instructor or not. A Lesson may define one or more learning activities. | LearningActivity |
| Lesson | A detailed description of the course of instruction for a ­short period of time that is used by a teacher to guide class instruction.  A Unit contains one or more lessons. | Lesson |
| Primary Source | An artifact, document, recording, or other source of information that was created at the time under study and provides first-hand testimony or direct evidence concerning a topic under investigation. | PrimarySource |
| Rubric/Scoring Guide | A document or guide that is used to delineate consistent criteria for grading. | RubricScoringGuide |
| Self Assessment | An assessment in which the user gathers information about and reflects on his or her own knowledge, skills, learning, or attitudes. | SelfAssessment |
| Text | The body of a printed work, to include reading passages. | Text |
| Textbook | A book used as a standard source of information on a particular subject. | Textbook |
| Unit | A long-range plan of instruction on a particular concept; it contains multiple lessons that are related. | Unit |
| Other |   | Other |


See also: 
Learning Resource Digital Media Type: https://ceds.ed.gov/CEDSElementDetails.aspx?TermxTopicId=26271 

Expand the "Learning Resources" folder here to see all of the related CEDS elements:
 https://ceds.ed.gov/domainEntitySchema.aspx

**Comment by michaelejay** [comment]
Hugh and Jim -

The challenge we had in refining the vocabulary had a lot to do with common usage vs technical usage of terminology and the need to describe legacy media. In several cases the medium and the type of resource are historically tied. When that list was initially developed, 'book' still largely conjured up images of a bound, printed resource whereas now more people are aware that a 'book' can be distributed in a variety of forms. We need to make sure we bridge common usage and semantically accurate usage to ensure the tools we create are usable by the largest audience, even as we educate those users about accurate use of the available terms.

- Michael

------------------------------------------
Michael Jay, President
Educational Systemics, Inc.

"Bringing the power of systemic thinking to the education enterprise."

Office: 360-752-2874 | Cell: 360-223-2453 | Fax: 443-347-2464

Join Michael and his education industry guests on Ed Table Talk with Michael Jay, a monthly show with lively discussion exploring controversial topics in the education industry.  Our 26 April show is titled:
           What’s left in the budget?: School need & purchases in the Spring
Subscribe: www.edtabletalk.org

**Comment by HughP** [comment]
Thank you for that answer. It is really helpful. I did find the list here in this repo. But I was still wondering what to do with 
* syllabus
and 
* handout

**Comment by philbarker** [comment]
Syllabus is also relevant to our discussion about courses [1], and I don't know where that will go. It may be that a syllabus is a subtype of CreativeWork type in schema.

Handout seems rather vague. Anything could be handed out. I would rather see it broken down to give more information about the nature of the handout (e.g. worksheet, information sheet, exemplar -- I don't know what he exact terms would be). But I think there is something there that could be added.


1. https://www.w3.org/community/schema-course-extend

**Comment by HughP** [comment]
@philbarker I agree that _handout_ is overly vague. But are you suggesting that your suggested terms should be added to the work being done at Schema-course-extended or to the Vocabulary at https://github.com/dcmi/LRMI_Vocabs/blob/master/learningResourceType.ttl?

It seemed that things like _Case-Studies_ (illustrative stories to be discussed or contemplated by the student) would be described with the term `text` in the vocabulary at: https://github.com/dcmi/LRMI_Vocabs/blob/master/learningResourceType.ttl 
Depending on the subject matter _Case-Studies_ might be analogous to what you have in mind for _ exemplar_. In a review of the kind of things in our collection which are labeled as _handout_ we have a lot of lecture summaries, or additional information which does not typically get discussed in lecture, but will be useful to the student. Some _handouts_ are also printouts of _power point presentations_. _Power point presentations_ or visual projected materials do not seem to have a clear classification either.

**Comment by philbarker** [comment]
@HughP I think we should discuss them for addition to the LRMI Vocab. I guess we should formalize what we have (we haven't yet formally published the existing list) and also think about what might be added.

There might be something similar to syllabus that arises from schema-course-extend, and so there needs to be some thought given to coordination.

**Comment by stuartasutton** [comment]
+1 The LRMI vocabulary in github has not yet been formally published; and
its further its development should be mindful of what is occuring in the
schema course extension as well as elsewhere in the education/training
ecosystem.

**Comment by HughP** [comment]
@philbarker 
Just an interesting note (since we both agree that the term _handout_ is ambiguous, and should likely be avoided): the prominent places to look up the LRMI specification all give the following clarifying note which includes the term _handout_ for Learning Resource Type: "For example, 'presentation', 'handout'."

* http://dublincore.org/dcx/lrmi-terms/
* http://schema.org/learningResourceType
* http://dublincore.org/dcx/lrmi-terms/1.1/

Whenever this vocabulary gets sorted, it would probably be great if the term _handout_ were not used as an example recommendation.

**Comment by tombaker** [comment]
@HughP This thread happened to catch my eye because of the version number `/1.1/` embedded in the URI `http://dublincore.org/dcx/lrmi-terms/1.1/`.

The first URIs for Dublin Core included such information (e.g., `http://purl.org/dc/elements/1.0/title`), but we decided in 2001 [against having namespace URIs that "carry versioning information"](http://dublincore.org/documents/dcmi-namespace/) because "embedding versioning information within the DCMI namespace URI was unlikely to be helpful to the long-term stability of DCMI namespace URIs or DCMI term URIs within those DCMI namespaces."  The policy distinguished between URIs for namespaces (and their terms) versus URIs for Web documents ("DCMI recommendations") and schema files ("DCMI term declarations") published about those terms: "it was felt that versioning information should be carried within the DCMI recommendations and/or DCMI term declarations associated with DCMI namespaces and terms."

The maintainers of the FOAF vocabulary also came to regret their decision to embed versioning information in an identifier for something that can, and should, evolve over time -- the URI for the FOAF property `maker` is still, to this day, `http://xmlns.com/foaf/0.1/maker`.

**Comment by philbarker** [comment]
@tombaker The namespace URI  for LRMI is http://dublincore.org/dcx/lrmi-terms/

Phil
